pkg/activator/handler/handler_test.go

```diff
@@ -124,6 +128,7 @@ func TestActivationHandler(t *testing.T) {
 			Config:     "config-real-name",
 			StatusCode: http.StatusOK,
 		}},
+		gpc: 1,
```

Comment:
do we want a test case with the probe disabled?

Reply:
Most of the tests are that.

Reply:
Also coverage of this code is basically 100%